aviatrix/utils.go

```diff
@@ -196,3 +200,97 @@ func DiffSuppressFuncGatewayVpcId(k, old, new string, d *schema.ResourceData) bo
 
 	return false
 }
+
+func reverseArray(arr []string) []string {
```

Comment:
Small nit: Technically should be reverseSlice

Reply:
have removed the reverseArray function